JavaScript/JavaScriptSDK/Context/User.ts

```diff
@@ -11,6 +11,11 @@ module Microsoft.ApplicationInsights.Context {
         public id: string;
 
         /**
+         * Authorized user id
+         */
+        public authId: string;
```

Comment:
I think this should be `private` if we need cookie to be updated when `authId` is updated.